docs/Compile.rst

```diff
@@ -81,11 +76,13 @@ You should be able to find them in your distro repositories (on Arch linux
 
 To build Stonesense, you'll also need OpenGL headers.
 
-Some additional dependencies for other distros are listed on the
-`wiki <https://github.com/DFHack/dfhack/wiki/Linux-dependencies>`_.
+On 64-bit Ubuntu, ``apt-get install zlib1g-dev:i386 libxml-libxml-perl libxml-libxslt-perl``.
+On 32-bit Ubuntu, ``apt-get install zlib1g-dev libxml-libxml-perl libxml-libxslt-perl``.
+Debian-derived distros should have similar requirements.
+
```

Comment:
These should probably be listed earlier, to avoid confusion with the stonesense dependencies mentioned above.